api/envoy/extensions/filters/http/aws_request_signing/v3/aws_request_signing.proto

```diff
@@ -35,6 +43,9 @@ message AwsRequestSigning {
   // endpoint.
   //
   // Example: us-west-2
+  //
+  // When using aws_sigv4a, the region specifier is now used as a region set. A region set is a comma separated list of
+  // AWS regions, such as ``us-east-1,us-east-2`` or wildcard ``*``
```

Comment:
Seems this comment is repeated, is this necessary.

Reply:
removed